tests/test_concurrency.py

```diff
@@ -34,18 +34,24 @@ def test_concurrent_conflict_uuid(dbos: DBOS) -> None:
     condition = threading.Condition()
     step_count = 0
     txn_count = 0
+    notified = False
 
     @DBOS.step()
     def test_step() -> str:
```

Comment:
Realizing that this is a test file of little consequence, this logic doesn't follow any threading pattern / also looks wrong.

Reply:
Can you elaborate it more? I had to add "notified" because if the second thread proceeds first, then it would notify but nobody is awaiting for it. Then the first thread would get stuck in `condition.wait()` forever.